app/views/teacher_training_adviser/steps/_subject_not_found.html.erb

```diff
@@ -4,7 +4,7 @@
       <div class="govuk-grid-column-two-thirds">
         <h1 class="govuk-heading-l">We're sorry, but you are not eligible for this service.</h1>
 
-        <p>To be eligible, you must be planning to teach maths, physics or modern foreign languages (MFL).</p>
+        <p>To be eligible, you must be planning to teach chemistry, computing, maths, modern foreign languages (MFL) or physics.</p>
```

Comment:
Thanks for adding this one in!

Do we need the `(MFL)`? We don't have this anywhere else?

NB I know this is on the original and not something you've added!

Reply:
good point, will take it out